Header header
int8 task_id
int8 robot_id
